### ISSUE TITLE
Zotero integration

### ISSUE THREAD

**Issue by parrenin** [ONLYOFFICE/plugin-zotero#19]
**Do you want to request a *feature* or report a *bug*?**
Feature
**What is the current behavior?**
OnlyOffice does not support the Zotero bibliography manager.
**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**

**What is the expected behavior?**
OnlyOffice should support the Zotero bibliography manager, to be successful in the academic world.
The Zotero people are apparently not interested to work on this:
https://forums.zotero.org/discussion/79081/zotero-pluggin-for-onlyoffice
**Which versions of DesktopEditors, and which OS are affected by this issue? Did this work in previous versions of DesktopEditors?**
All versions.

**Comment by menelic** [comment]
Full zotero support as in Google Docs is key for Only Office to offer a true FOSS replacement of corporate cloud and collaboration infrastructures. The ability to draw from the zotero database for inline source citations that are automatically appended to the ordered and pre- formatted aoutogenerated bibliograhy is the main draw of zotero. This feature would unlock full academic collaborative writing.

**Comment by QuietOcean** [comment]
Could somebody explain what step might be required that the Zotero plugin is fully functional (with citation and Webdav support), much like LibreOffice? I cannot help to feel that we are getting very close... but I have what is needed to make this work.

**Comment by VEZY** [comment]
> Full zotero support as in Google Docs is key for Only Office to offer a true FOSS replacement of corporate cloud and collaboration infrastructures. The ability to draw from the zotero database for inline source citations that are automatically appended to the ordered and pre- formatted aoutogenerated bibliograhy is the main draw of zotero. This feature would unlock full academic collaborative writing.

I too am only waiting for a better integration of Zotero to completely switch to OnlyOffice. I think it is the only drawback of this suite at the moment. And I'm sure a lot of academics would switch too if it was possible to have the References listed at the end of the document as in MS Word, and manage them after addition (_e.g._ remove, change the citation style...).

**Comment by Nationalgeographic** [comment]
> > Full zotero support as in Google Docs is key for Only Office to offer a true FOSS replacement of corporate cloud and collaboration infrastructures. The ability to draw from the zotero database for inline source citations that are automatically appended to the ordered and pre- formatted aoutogenerated bibliograhy is the main draw of zotero. This feature would unlock full academic collaborative writing.
> 
> I too am only waiting for a better integration of Zotero to completely switch to OnlyOffice. I think it is the only drawback of this suite at the moment. And I'm sure a lot of academics would switch too if it was possible to have the References listed at the end of the document as in MS Word, and manage them after addition (_e.g._ remove, change the citation style...).

Yes. Would also completely switch to OnlyOffice, when Zotero is integrated the way it should (like in Word(!) or LibreOffice).To me the key features are the citation and the automated bibliography, both used from my local zotero client/database; without those features, I can not write anymore :)

**Comment by TS-CUBED** [comment]
Second that. 

Full Zotero integration would be the one thing that would make me switch all my students over to OnlyOffice

**Comment by kbundg** [comment]
Lets have it then! :+1:

**Comment by shu0406** [comment]
I would like to have at least the option "insert citation" additionally to "insert bibliography". That would be a first step. 
![grafik](https://user-images.githubusercontent.com/76654412/151660607-a27fc5e2-daa0-4aa4-9e72-bbad77d074a0.png)

**Comment by dagdrommer** [comment]
I agree with the others. The lack of a well integrated Citation-Manager, especially Zotero, discourages the academic world from switching to OnlyOffice.

**Comment by ShockwaveNN** [comment]
So this issue is a little mess from different problems so I'll clarify this

Currently, we got issue 50849 in our private issue tracker for the ability to add Zotero citations and full bibliography support. Right now there is not a lot of traction in implementing it, but you can always contact sales@onlyoffice.com and discuss our priorities

Issue 43716 mentioned earlier was about `Making plugin with some feature set`, so it was closed as soon as the plugin was released
I'll rename this issue to be more clear

**Comment by Rosmaninho** [comment]
This feature would be essential for OnlyOffice maybe to even replace LibreOffice as the premier open-source office suite IMO.
It needs fully functional citation manager plugins. And Zotero is the leading open-source citation manager. You guys should approach the Zotero devs now that they got Zotero 6 out of the way.

**Comment by dekipantelija** [comment]
I must admit it is frustrating that none of the 3 available citation helpers are available in desktop editor. I do have an online version available with Infomaniak, but they won't allow inclusion of plugins. As much as i am ok paying the 150 to get to your servers, i still feel weird paying that amount to get one feature more.

**Comment by ShockwaveNN** [comment]
@dekipantelija Hi, your message seems unrelated to OP issue (about bibliography and citation)

Zotero is not included by default in DesktopEditors, but can be installed via this instruction in [README](https://github.com/ONLYOFFICE/plugin-zotero#how-to-install-to-desktop)

If you got any troubles not related to bibliography/citation feature - please create a new issue

**Comment by tvrbanec** [comment]
I attend to completely switch to OnlyOffice when the Zotero plugin gets full integration in it. And that would be the last removed obstacle to completely stopping using Windows OS. When writing academic articles, I must use virtual Windows because of the need for Word (LibreOffice Writer is very good, but it is not compatible enough with Word).

**Comment by jdpipe** [comment]
Great work on this plugin so far. However adding support for citations is an essential next step before this plugin is genuinely useful for academic writing. May I suggest that the README for this plugin more explicitly mention this shortcoming -- it's basically still a work in progress -- as it may help to attract the needed contributions.

**Comment by ShrewMaster** [comment]
I came here to add up to the pile of comments requesting support for citations. This is a really important feature in the academic world. Scientists that wish to transition to a free and open-source alternative to Microsoft Office can't switch to OnlyOffice as long as the Zotero plugin is lacking a citation feature. It's a real shame because the other Office alternatives -- Microsoft Office, LibreOffice, even Google Docs! -- have this feature implemented, and currently provide better alternatives for academics to write manuscripts.

Please make this happen, OnlyOffice team. We really want to change to your software!
